lerna.json

```diff
@@ -1,4 +1,4 @@
 {
-  "packages": ["*"],
+  "packages": ["*", "firestore-translate-text/functions"],
```

Comment:
Note: This could be `"*/functions"` but this then this ends up including the BQ functions - can switch to `"*/functions"` once BQ extension `package.json` & tests have been refactored.

Reply:
sounds good